core-sitemaps.php

```diff
@@ -25,51 +25,56 @@
  * Version:           0.3.0
  */
 
+// Bail early if Sitemaps is already defined. Prevents plugin from loading after core merge.
+if ( class_exists( 'Sitemaps' ) ) {
```

Comment:
I think most of the logic in `Core_Sitemaps` will look much different in a core patch, perhaps even go away. We should check for another class here. Or maybe even just a function as a lower hanging fruit.

Reply:
ok, I'll change to a function check for now and add a comment to revisit for the merge patch to make sure we keep it working.

Reply:
changed to a function_exists check.